project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.module

```diff
@@ -76,6 +77,24 @@ function _c4m_message_format_welcome_to_group_from_field(&$message, $message_ent
 }
 
 /**
+ * Formats 'c4m_new_membership_request' message From field, per requirement.
+ */
+function _c4m_message_format_new_membership_request_from_field(&$message, $message_entity) {
+  if ($message_entity->type != 'c4m_new_membership_request') {
+    // For message type other than c4m_new_membership_request - return early.
+    return;
+  }
+
+  $message['from'] = format_string(
+    '"The @site team"<@mail>',
+    array(
+      '@site' => variable_get('site_name'),
+      '@mail' => 'capacity4dev@ec.europa.eu',
```

Comment:
There're many hard coded mail values at template document.
See MT48, for example.